internal/v5/api.go

```diff
@@ -994,6 +1009,7 @@ func checkExtraInfoKey(key string, field string) error {
 // GET id/meta/perm
 // https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetaperm
 func (h *ReqHandler) metaPerm(entity *mongodoc.BaseEntity, id *router.ResolvedURL, path string, flags url.Values, req *http.Request) (interface{}, error) {
```

Comment:
Thinking about this update, does it now mean that if I attempt to update the development ACLs using meta perm I can only do it if there is already an entity published in the channel? Or what does PUT /v5/~charmers/wordpress/meta/perm?channel=stable do if there is no wordpress in a stable channel?